pkgs/development/python-modules/llama-cpp-python/default.nix

```diff
@@ -49,7 +49,8 @@ buildPythonPackage rec {
 
   dontUseCmakeConfigure = true;
   SKBUILD_CMAKE_ARGS = lib.strings.concatStringsSep ";" (
-    lib.optionals cudaSupport [
+    [ "-DGGML_NATIVE=off" ]
```

Comment:
did't realise that would use -march=native 

https://github.com/ggerganov/llama.cpp/pull/11366/files

Reply:
btw I think this issue may have exposed the problem in `cc-wrapper` where we filter out `-mcpu=native` but not anything with features (set with `+(no)<...>`. It may be one source of irreproducibility, since apparently `clang` swallows these modified feature flags (and probably produces optimized binaries).

Reply:
One other idea that we may want to adopt in our toolchain is setting that `SOURCE_DATE_EPOCH` variable since, apparently, it's well known / standardized over other projects that attempt reproducible builds. I'll take a look later into both issues.